docs/src/main/paradox/routing-dsl/path-matchers.md

```diff
@@ -216,16 +216,16 @@ Path matchers can be combined with these combinators to form higher-level constr
 
 Tilde Operator (`~`)
 : The tilde is the most basic combinator. It simply concatenates two matchers into one, i.e if the first one matched
-(and consumed) the second one is tried. The extractions of both matchers are combined type-safely.
+(and consumed) the second one is tried. The extractions of both matchers are combined type-safely. (This is an alias for the `append` method.)
 For example: `"foo" ~ "bar"` yields a matcher that is identical to `"foobar"`.
 
 Slash Operator (`/`)
-: This operator concatenates two matchers and inserts a `Slash` matcher in between them.
+: This operator concatenates two matchers and inserts a `Slash` matcher in between them. (This is an alias for the `slash` method.)
 For example: `"foo" / "bar"` is identical to `"foo" ~ Slash ~ "bar"`.
```

Comment:
```suggestion
For example: `"foo" / "bar"` is identical to `"foo" slash "bar"`. You can also use the composition of `~` and path mathcers so `"foo" ~ Slash ~ "bar"` is also equivalent.
```

Reply:
Yeah, listing all the alternatives may be too much. Let's keep it as you originally posted.